eng/Version.Details.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.19463.1">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.19474.3">
```

Comment:
This doesn't appear to be on the dotnet-core feed. Is it somewhere else?

Reply:
@mmitche is this expected to work? My understanding is that we want this because it fixes publishing, right?